cypress/e2e/pages/articles/tests.js

```diff
@@ -61,179 +44,80 @@ export const testsThatFollowSmokeTestConfig = ({
       });
     });
 
-    describe(`Article Body`, () => {
-      it('should render a H1, which contains/displays a styled headline', () => {
-        const headlineData = getBlockData('headline', articlesData);
-        cy.get('h1').should(
-          'contain',
-          headlineData.model.blocks[0].model.blocks[0].model.text,
-        );
-      });
```

Comment:
Note for reviewers: this has been deleted because it's checking the content of a H1, which is already covered by an integration test - https://github.com/bbc/simorgh/blob/fdc0bb7fc211b92dc6f7bada966e5682089415c9/src/integration/common/mainHeading.js